File: Python/timeOfDay.py
import datetime

now = datetime.datetime.now().hour

print(now)

if (now <=11):
    print("Good Morning")
elif (now <=16):
    print("Good afternoon")
elif (now <=20):
    print("Good Evening")
else:
    print("GOOD NIGHT. GO TO BED.")

File: C/timeOfDay.c
#include <stdio.h>
#include <time.h>

int hour;


int main(void){

    time_t now;
    struct tm*now_tm;

    now = time(NULL);
    now_tm = localtime(&now);
    hour = now_tm->tm_hour;
    printf("%d\n", hour);

    if (hour <= 12){
        printf("Good Morning.\n");
    }else if (hour <= 18){
        printf("Good Afternoon.\n");
    }else if (hour <= 20){
        printf("Good Evening.\n");
    }else{
        printf("Good NIGHT. GO TO SLEEP NOW.\n");
    }

    
    return 0;
}